clients/scanoss/src/main/kotlin/model/Model.kt

```diff
@@ -68,6 +68,21 @@ data class License(
     val osadlUpdated: String? = null
 )
 
+/**
+ * The “copyrights” section of the RAW report.
+ */
+@Serializable
+data class Copyright(
+    /** Copyright found. */
+    val name: String,
+
+    /**
+     * Location where the copyright was mined from.
+     * Please note that the copyright cannot have the value [LicenseSource.FILE_SPDX_TAG]; we kept a single enum for
```

Comment:
Please continue the sentence in the first line until it needs to be wrapped at column 120.